remote/signature.go

```diff
@@ -35,7 +35,10 @@ func sigParams(pk crypto.PrivKey, ref dsref.Ref) (map[string]string, error) {
 		"profileID": ref.ProfileID,
 		"path":      ref.Path,
 
-		"pid":       pid,
+		"pid": pid,
+		// subject is the client node's username, will be used
+		// on the remote side to determine access control
+		"subject":   subject,
```

Comment:
I think this should be `username`. The entire map is talking about the `subject`

Reply:
Ahh, yes, however, we are also transmitting the `ref` over, which has the field `username`. Can change to `subject_username` so it is more clear?

Reply:
ugh, sorry, PR preview cut the rest of this map off. let's use `requester` whenever we're talking about who is making the request, and pass _requester_ values as the "subject" for access control enforcement